src/redux/missions/missions.js

```diff
@@ -0,0 +1,12 @@
+// const URL = 'https://api.spacexdata.com/v3/missions';
+
+const initialState = [];
+export default function reducer(state = initialState, action) {
+  const { type } = action;
+  switch (type) {
+    // case GET_MISSIONS:
+    //   return payload.missions;
```

Comment:
Comments could be deleted as it makes your code look unorganized. 👍

Reply:
Hi @AdrianCSM  thanks for your approval .